spec/features/debates_spec.rb

```diff
@@ -353,9 +360,9 @@
   feature 'Debate index order filters' do
 
     scenario 'Default order is hot_score', :js do
-      create(:debate, title: 'Best').update_column(:hot_score, 10)
-      create(:debate, title: 'Worst').update_column(:hot_score, 2)
-      create(:debate, title: 'Medium').update_column(:hot_score, 5)
+      create(:debate, participatory_process: participatory_process, title: 'Best').update_column(:hot_score, 10)
```

Comment:
I'm sure there's a way to DRY this up. Maybe we could create a DSL somehow:

```ruby
with_participatory_process(participatory_process) do
  create(:debate, title: 'Worst')
end
```

As `create` would be executed in the context of the block, we could capture the method and re-write it to include the participatory process:

```
def create(model, options = {})
  create(model, { participatory_process: participatory_process }.merge(options))
end
```